frontend/src/Maker.tsx

```diff
@@ -111,11 +127,36 @@ export default function App() {
                             >
                                 {order ? "Update Sell Order" : "Create Sell Order"}
                             </Button>
-                            {order && <OrderTile order={order} />}
                         </VStack>
                     </VStack>
                 </GridItem>
+                <GridItem colStart={3} colSpan={2}>
+                    {order && <OrderTile order={order} />}
+                </GridItem>
             </Grid>
+            <Tabs>
+                <TabList>
+                    <Tab>Running [{running.length}]</Tab>
+                    <Tab>Open [{open.length}]</Tab>
+                    <Tab>Closed [{closed.length}]</Tab>
+                    <Tab>Unsorted [{unsorted.length}] (should be empty)</Tab>
```

Comment:
Why does this column exist, what is it for? Can we just remove it if it's always empty?

Reply:
It's only _temporarily_ to detect if backend and frontend get out of sync in regards to the CFD states. E.g. if we introduce a new _running_ state in the backend but not in the frontend we might wonder why the CFD does not show up under `running`. It will then show up in `unsorted` and requires adaptation. Eventually this tab should disappear.